src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,344 @@
+    private Y = 0
+    private boxSizeX = 800
+    private boxSizeY = 90
+    //private boxSize = this.sketch.createVector(800, 90)
+    private canvasSizeX = 800
+    private canvasSizeY = 800
+    private subG = this.sketch.createGraphics(this.boxSizeX, this.boxSizeY)
+    private subL = this.sketch.createGraphics(
+        this.canvasSizeX,
+        this.canvasSizeY
+    )
+    private boxIsShow = false
+    private primeNum: number[] = []
+    private countPrime = 0
+    private firstDraw = true
+    private showLabel = false
+    private brightAdjust = 100
+
+    initialize(sketch: p5, seq: SequenceInterface) {
+        super.initialize(sketch, seq)
+        this.sketch = sketch
+        this.seq = seq
+        this.currentNum = seq.first
+        this.X = 0
+        this.Y = 0
+        this.ready = true
+    }
+
+    setup() {
+        this.sketch.background('black')
+        this.sketch.colorMode(this.sketch.HSB, 360, 100, 100)
+        this.sketch.frameRate(30)
+
+        this.firstDraw = true
+
+        // Set position of the circle
+        this.X = this.canvasSizeX / this.n + 30
+        this.Y = this.canvasSizeX / this.n + 50
+
+        // Obtain all prime numbers from the sequence
+        for (let i = this.seq.first; i < this.n; i++) {
+            const checkCurrentPrime = Number(this.seq.getElement(i))
+            if (
+                this.isPrime(checkCurrentPrime)
+                && !this.primeNum.includes(checkCurrentPrime)
+            ) {
+                this.primeNum.push(checkCurrentPrime)
+                this.countPrime += 1
+            }
+        }
+    }
+
+    draw() {
+        if (this.firstDraw == true && this.currentNum < this.n) {
+            const currentElement = Number(
+                this.seq.getElement(this.currentNum++)
+            )
+            this.drawCircle(currentElement)
+
+            this.changePosition()
+
+            // Check if drawing finished
+            if (this.currentNum >= this.n) {
+                this.firstDraw = false
+            }
+            this.sketch.noStroke()
+        } else {
+            // Monitor keyboard events after finishing drawing
+            // Bug: multiple clicks detected when only one click happened
+            this.keyboardEvents()
+        }
+    }
+
+    keyboardEvents() {
+        // Show description box when the left arrow key is pressed
+        if (this.sketch.keyIsDown(this.sketch.LEFT_ARROW)) {
+            if (this.boxIsShow == true && this.subG != null) {
+                this.boxIsShow = false
+                this.undrawBox()
+            } else {
+                this.boxIsShow = true
+                this.drawBox()
+            }
+        }
+
+        // Show label when right arrow key is pressed
+        if (
+            this.sketch.keyIsDown(this.sketch.RIGHT_ARROW)
+            && this.firstDraw == false
+        ) {
+            if (this.showLabel == false) {
+                this.showLabel = true
+                this.drawLabel()
```

Comment:
The visualizer will hang pretty quickly on very large numbers (of which there are plenty in the OEIS). You might want to have a cut-off for a, where if you see a number bigger than that cutoff, you just "fake it" by drawing a pretty non-fuzzy bright circle (I'm assuming that as the numbers get very big, the eye won't detect the "fake" here; maybe I'm wrong?).

Reply:
In a recent commit, I made it so the user can decide the growth function by formula, which should alleviate this (and besides, now it would be very difficult to do this).  So I would like to simply resolve this conversation -- no action suggested.  Sound ok?

Reply:
OK, resolving.